src/extensions/google.ts

```diff
@@ -0,0 +1,17 @@
+import * as BigQuery from '@google-cloud/bigquery'
+
+type DummyCloud = {
+    bigquery: typeof BigQuery
+}
+
+type DummyGoogle = {
+    cloud: DummyCloud
+}
+
+export function createGoogle(): DummyGoogle {
```

Comment:
Wow, people invested millions of dollars to create Google and here we are, doing the same with just one function!

Reply:
rookies